.github/workflows/release.yml

```diff
@@ -71,18 +71,12 @@ jobs:
           cache-dependency-path: |
             requirements.txt
             setup.py
-      - name: Install dependencies
-        run: |
-          pip install --upgrade pip wheel
-          pip install -r requirements.txt
-          pip install -e .
       - name: Release
         env:
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
         run: |
-          pip install setuptools wheel twine
+          pip install --upgrade pip wheel setuptools twine
           python setup.py sdist bdist_wheel
-          twine upload --skip-existing dist/*.whl
-          twine upload --skip-existing dist/*.tar.gz
+          twine upload --skip-existing dist/*.{whl,tar.gz}
```

Comment:
@shyuep One thing I'm unsure about is if this bash expansion works on windows:

```sh
dist/*.{whl,tar.gz}
```

If not, we could just do

```sh
dist/*
```

Reply:
I wouldn't bother with the expansion. It is not critical to the functioning of the code. Technically, only the tar.gz needs the skip-existing arg. This is the non-compiled version that can get uploaded several times and the workflow will fail if skip-existing is not used. The wheels for each OS and python version should be unique. But we can just leave it as is.